distribution/lib/Standard/AWS/0.0.0-dev/src/S3/S3.enso

```diff
@@ -18,16 +18,10 @@ import project.S3.S3_File.S3_File
 polyglot java import java.io.IOException
 polyglot java import org.enso.aws.BucketLocator
 polyglot java import org.enso.aws.ClientBuilder
+polyglot java import org.enso.aws.file_system.S3Utils
 polyglot java import software.amazon.awssdk.core.exception.SdkClientException
-polyglot java import software.amazon.awssdk.services.s3.model.CopyObjectRequest
-polyglot java import software.amazon.awssdk.services.s3.model.DeleteObjectRequest
-polyglot java import software.amazon.awssdk.services.s3.model.GetObjectRequest
-polyglot java import software.amazon.awssdk.services.s3.model.HeadBucketRequest
-polyglot java import software.amazon.awssdk.services.s3.model.HeadObjectRequest
```

Comment:
Replacing all these imports with S3Util is good from code perspective, generated image size, maintainability & co. We should use this “wrapper model” more in other “polyglot Java import” cases.